apps/communications/contacts/js/views/list.js

```diff
@@ -1715,9 +1715,8 @@ contacts.List = (function() {
         callback(uuid);
       });
     }
-    }
-    evt.preventDefault();
```

Comment:
Good :+1: 
and this change allows us to find one more indentation error: line 1720 - function's closing curly brace has obviously wrong indentation, this happened due to wrong indentation of code block starting from line 1700, so hopefully last update required, please indent lines 1700-1720 by 2 'spaces'